src/gradata/integrations/langchain_adapter.py

```
@@ -110,4 +110,4 @@ def save_context(self, inputs: dict, outputs: dict) -> None:
 
     def clear(self) -> None:
         """No-op for persistent brain memory."""
-        pass
\ No newline at end of file
+        pass
```

Ruff (0.15.9)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)